tests/unit/lib/build_module/test_build_strategy.py

```diff
@@ -39,14 +39,22 @@ def setUp(self):
         self.build_graph.put_layer_build_definition(self.layer_build_definition2, Mock())
 
 
+class _TestBuildStrategy(BuildStrategy):
+    def build_single_function_definition(self, build_definition):
+        return {}
+
+    def build_single_layer_definition(self, layer_definition):
+        return {}
+
+
 class BuildStrategyTest(BuildStrategyBaseTest):
     def test_build_functions_layers(self):
-        build_strategy = BuildStrategy(self.build_graph)
+        build_strategy = _TestBuildStrategy(self.build_graph)
```

Comment:
would you explain why we need this  "_TestBuildStrategy" class?

Reply:
`BuildStrategy` is an abstract class so I need to create a subclass to instantiate an object.

Reply:
(and adding prefix `_` to indicate it is a "private" class)